datastore/shared/postgresql_backend/pg_connection_handler.py

```diff
@@ -97,8 +97,12 @@ def __init__(self, shutdown_service: ShutdownService):
         self.sync_event = Event()
         self.sync_event.set()
 
-        min_conn = int(self.environment.try_get("DATASTORE_MIN_CONNECTIONS") or 1)
-        max_conn = int(self.environment.try_get("DATASTORE_MAX_CONNECTIONS") or 1)
+        min_conn = max(
+            int(self.environment.try_get("DATASTORE_MIN_CONNECTIONS") or 2), 2
```

Comment:
Maybe throw an error here if `DATASTORE_MIN_CONNECTIONS` is lower than 2, since then the datastore cannot work correctly

Reply:
You cannot set the value lower than 2, because the statement forces a minimum value of 2. Background: The dev-compose-yaml's don't set the connections at all. Before  this, they assumed a 1, now the default is 2 for he number of db-connections.
But we should separate this parameter, because in practice the reader, in combination with gunicorn-thread-settings should have a higher value, the writer as it is, always 2.

Reply:
Ah, I missed the `max`, my bad